### PR TITLE
fix(wallet): creating account with string key file

### PR DESCRIPTION
**Pull request by Pavneet-Sing**
Resolves https://github.com/brave/brave-browser/issues/19971

## Summary
We do allow account creation by importing a string key from a file so it has been fixed by trimming the newline char and no need to update the hint.

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by Pavneet-Sing** [comment]
I think we should disable the JSON key EditText (or change it to text) because user cannot add an account by typing the key(or pasting).

**Review by deeppandya — APPROVED** [review]
LGTM

File: browser/ui/android/strings/android_brave_strings.grd [hunk 1]
```diff
@@ -2135,7 +2135,7 @@ Are you sure you want to do this?
       You can create a secondary account by importing your private key.
     </message>
     <message name="IDS_IMPORT_ACCOUNT_TEXT" desc="Brave wallet accounts text">
-      Enter, paste, or import your private key string file or JSON.
+      Import from a JSON file only.
```
Comment (SergeyZhukovsky) [inline browser/ui/android/strings/android_brave_strings.grd:2138]: that's not right as private key import works. How do you suppose to import from a private key?

Comment (Pavneet-Sing) [reply browser/ui/android/strings/android_brave_strings.grd:2138]: gonna study the behaviour from all platforms and will try to reproduce the failed case then will update the PR and current UI accordingly.

**Review by SergeyZhukovsky — APPROVED** [review]
++

**Pull request merged**